src/main/java/com/sparta/spangeats/domain/store/controller/StoreController.java

```diff
@@ -1,4 +1,46 @@
 package com.sparta.spangeats.domain.store.controller;
 
+import com.sparta.spangeats.domain.member.entity.Member;
+import com.sparta.spangeats.domain.store.dto.ErrorResponseDto;
+import com.sparta.spangeats.domain.store.dto.StoreRequestDto;
+import com.sparta.spangeats.domain.store.dto.StoreResponseDto;
+import com.sparta.spangeats.domain.store.service.StoreService;
+import com.sparta.spangeats.security.filter.UserDetailsImpl;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.apache.tomcat.util.net.openssl.ciphers.Authentication;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/stores")
+
 public class StoreController {
+
+    @Autowired
+    private StoreService storeService;
```

Comment:
어차피 `@Autowired`를 붙이지 않아도 자동으로 의존성을 추가해줍니다! 어차피 서영님께서 생성자로 의존성 주입을 해주었기 때문이죠!! `@RequiredArgsContructor` 어노테이션이 `final` 이 붙은 멤버 변수들을 매개변수로 생성자를 만들어주거든용

따라서 저 `@Autowired`를 빼고, `StoreService storeService`에 `final`을 붙여주시면 됩니다~`@Autowired`는 지양해주세용! 이유는 블로그 [첨부해드리겠습니다.](https://dazbee.tistory.com/43)
```suggestion 
    private final StoreService storeService;
```